### PR TITLE
Add Edit Button to documentation pages

### PR DESCRIPTION
**Pull request by flower-of-the-bridges**
## Description

This pull request introduces the `Edit Page` button provided by the [docusaurus content docs plugin](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#editUrl).

This feature is enabled only on pages that are not auto-generated or not of old versions. In particular, the following resources are excluded:

* marketplace plugins;
* marketplace applications;
* back-kit backoffice library;
* older documentation versions.

## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [x] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

### CODE REVIEW

File: createEditUrl.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**  list of resources that does not need to be edited */
+const UNEDITABLE_RESOURCES = [ 
+  'release-notes',
```
Comment (fredmaggiowski) [inline createEditUrl.js:3]: why not the release notes?

**Comment by fredmaggiowski** [comment]
Nice, thanks!

Can you submit a screenshot of how the button appears?

File: createEditUrl.js (hunk 1 above)
Comment (flower-of-the-bridges) [reply createEditUrl.js:3]: Hi @fredmaggiowski I thought that the release notes were generated automatically by a changelog. If this is not the case, I can remove it

**Comment by flower-of-the-bridges** [comment]
Hi @fredmaggiowski 

This is how the page appears with the button:

image

**Review by fredmaggiowski — APPROVED** [review]
LGTM

@akelity can you review it as well?

**Comment by akelity** [comment]
> LGTM
> 
> @akelity can you review it as well?

What happens if someone makes changes to the pages managed by the doc aggregator? Should they be excluded from editing?

I checked and the modification does not exclude these pages.

Basically all the page inside the folders:

- docs/runtime_suite
- docs/runtime_suite_examples
- docs/runtime_suite_templates
- docs/runtime_suite_libraries
- docs/runtime_suite_tools
- docs/business_suite

**Pull request merged**